test/helpers.lua

```diff
@@ -15,7 +15,13 @@ local function shell_quote(str)
   end
 end
 
-local function argss_to_cmd(...)
+local REMOVE_THIS = {}
+
+local global_helpers = {
```

Comment:
should just be `module`.

Reply:
..but `ctags` (universal-ctags) would use `module.eq` then instead of `global_helpers.eq`.
(the parser is very basic, and only recognized functions - it could in theory for example handle `-- @module global_helpers` (ldoc hint))
Anyway, do you still prefer `module` then?

Reply:
ctags does not drive our choices, and in any case the function names are usually quasi-unique. I've never had a problem with e.g.:

    :tag argss_to_cmd

`module` is a good pattern and helps new devs understand good practices and the intent.

Also ctags won't be needed once we have LSP and/or treesitter helping us.